src/parser.py

```diff
@@ -0,0 +1,201 @@
+from __future__ import annotations
+import requests as rq
+import json
+from bs4 import BeautifulSoup as bs
+import configuration as cf
+from transformers import pipeline
+
+question_name = "deepset/roberta-base-squad2"
+question_model = pipeline(
+    "question-answering", model=question_name, tokenizer=question_name
+)
+
+
+translation_name = "Helsinki-NLP/opus-mt-ru-en"
+translation_model = pipeline(
+    "translation", model=translation_name, tokenizer=translation_name
+)
+
+
+def pages_num(soup: bs) -> int:
+    pagination = soup.find("div", class_="PaginationWidget__wrapper-pagination")
+    paginator = pagination.find_all("a")
+    num = 0
+    for data_page in paginator:
+        if num < int(data_page["data-page"]):
+            num = int(data_page["data-page"])
+    return num
+
+
+def parce(to_file: str) -> None:
```

Comment:
`parse`

Reply:
And it is not necessary to specify `None` in the return type hint. AFAIK, in this case, it can be just omitted.

Reply:
parce -> parse  
The problem is its note from mypy. So i disable it option in ```setup.cfg```
![image](https://user-images.githubusercontent.com/90473864/207123868-9b1b977b-35da-4b4c-a2d2-633527b72354.png)
I thouht main purpose of mypy is in ```[no-untyped-def]```